charts/aws-ebs-csi-driver/templates/node.yaml

```diff
@@ -160,15 +142,15 @@ spec:
       volumes:
         - name: kubelet-dir
           hostPath:
-            path: /var/lib/kubelet
+            path: {{ .Values.node.kubeletPath }}
```

Comment:
windows support will be coming soon with a windows-specific daemonset. In which case, I think I'll need to add windows variation of this .Values.node.kubeletPath, unless you see a cleaner solution.

Reply:
We could maybe pull it into a helper template or something.  Let me know when that rolls around and we can look at the implications then.  Some of it depends on how different the daemonsets are from a kube perspective.  I assume someone might want to have both daemonsets running in a single cluster.

Reply:
Yes I plan to install both daemonsets by default so if someone has no windows nodes then the daemonset will simply have 0 pods.

There's a preview in my PR here, the main issue is kubeletPath has to be in windows format like C:\var\lib\kubelet, so simply copying the linux daemonset kubeletPath value won't work : https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/823/files#diff-de20e61e469f4cce4b8334c55256b2b593e91ae9a78fb4c6f0f52a357d695eefR63 Otherwise, I think it would make sense to copy all the other values since the daemonsets otherwise look similar.

Anyway, wanted to give a heads up, we can discuss it in detail in https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/823 later, it's still WIP and I'll end up rebasing on this PR nyway